<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edittimedialog</class>
 <widget class="QDialog" name="edittimedialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Date and Time Editor</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="cancelbutton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finishbutton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish and Apply</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioISOT">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>27</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit by ISOT</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioJD">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>57</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit by JD</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioDuration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>87</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit by Duration</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditISOT">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>27</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditJD">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>57</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDuration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>87</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelISOT">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>11</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
    </font>
   </property>
   <property name="text">
    <string>yyyy-mm-ddThh:mm:ss.ssssss</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>radioISOT</tabstop>
  <tabstop>radioJD</tabstop>
  <tabstop>radioDuration</tabstop>
  <tabstop>lineEditISOT</tabstop>
  <tabstop>lineEditJD</tabstop>
  <tabstop>lineEditDuration</tabstop>
  <tabstop>finishbutton</tabstop>
  <tabstop>cancelbutton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
